root/static/scripts/edit/URLCleanup.js

```diff
@@ -1227,17 +1227,18 @@ const CLEANUPS = {
     match: [new RegExp("^(https?://)?([^/]+\\.)?(nicovideo\\.jp/)","i")],
     type: _.default({}, LINK_TYPES.streamingmusic, LINK_TYPES.videochannel),
     clean: function(url) {
-      var m = /^(?:https?:\/\/)?(?:[^\/]+\.)?nicovideo\.jp\/(user\/[0-9]+|watch\/sm[0-9]+).*$/, "https://www.nicovideo.jp/$1";
+      return /^(?:https?:\/\/)?(?:[^\/]+\.)?nicovideo\.jp\/(?:(user)\/[0-9]+|(watch)\/sm[0-9]+).*$/, "https://www.nicovideo.jp/$1";
```

Comment:
Changes inside the matching pattern `/…/` were suggested for the `validate` function only. It won't work for the `clean` function as it relies on a single capturing group with `$1` in the replacement string. Please revert these.